### ISSUE TITLE
GET /api/v3/flows/executor/default-authentication-flow/

### ISSUE THREAD

**Issue by dieseldan** [goauthentik/authentik#7319]
**Describe the bug**
Unable to login or authenticate to Authentik after to upgrade to 2023.10.1. 

**To Reproduce**
1. Access Authentik via local IP or URL, or any application URL configured for authentication to Authentik
2. Login with any user, including akadmin user
3. Receive the message "Something went wrong! Please try again later." with a random Request ID when trying to access Authentik or any proxied applications. 

**Expected behavior**
Access and applications had been working with existing configuration as expected until upgrade to 2023.10.1. 

**Logs**
<details>
    <summary>Stacktrace from authentik</summary>

```
Traceback (most recent call last):
  File "/authentik/flows/views/executor.py", line 287, in get
    stage_response = self.current_stage_view.dispatch(request)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator_validate/stage.py", line 222, in get
    challenges = self.get_device_challenges()
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator_validate/stage.py", line 157, in get_device_challenges
    user_devices = list(devices_for_user(self.get_pending_user()))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator/__init__.py", line 93, in devices_for_user
    yield from device_set
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/models/query.py", line 398, in __iter__
    self._fetch_all()
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "/ak-root/venv/lib/python3.11/site-packages/sentry_sdk/integrations/django/__init__.py", line 628, in execute
    return real_execute(self, sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django_prometheus/db/common.py", line 69, in execute
    return super().execute(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/psycopg/cursor.py", line 737, in execute
    raise ex.with_traceback(None)
django.db.utils.ProgrammingError: relation "authentik_stages_authenticator_static_staticdevice" does not exist
LINE 1: ...tic_staticdevice"."throttling_failure_count" FROM "authentik...
                                                             ^
```
</details>

**Version and Deployment (please complete the following information):**
- authentik version: 2023.10.0
- Deployment: Docker on Unraid

**Additional context**
Following upgrade, receive the message "Something went wrong! Please try again later." with a random Request ID when trying to access Authentik or any proxied applications.
Downgraded back to 2023.8 by manually setting version tag and logins work again as expected.

**Comment by Demigodrick** [comment]
I'm getting exactly the same following the upgrade too - word for word the same log error:
<details>
`Traceback (most recent call last):
  File "/authentik/flows/views/executor.py", line 287, in get
    stage_response = self.current_stage_view.dispatch(request)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator_validate/stage.py", line 222, in get
    challenges = self.get_device_challenges()
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator_validate/stage.py", line 157, in get_device_challenges
    user_devices = list(devices_for_user(self.get_pending_user()))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator/__init__.py", line 93, in devices_for_user
    yield from device_set
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/models/query.py", line 398, in __iter__
    self._fetch_all()
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django_prometheus/db/common.py", line 69, in execute
    return super().execute(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/psycopg/cursor.py", line 737, in execute
    raise ex.with_traceback(None)
django.db.utils.ProgrammingError: relation "authentik_stages_authenticator_static_staticdevice" does not exist
LINE 1: ...tic_staticdevice"."throttling_failure_count" FROM "authentik...
                                                             ^`
</details>

**Comment by BeryJu** [comment]
This is fixed in 2023.10.1

**Comment by dieseldan** [comment]
> This is fixed in 2023.10.1

Confirmed. Upgraded to 2023.10.1 and it's working as expected.

**Comment by lightweight** [comment]
Hmm - I've just done an upgrade from 2023.6.1 to 2023.10.1 and I'm getting the following exception: 

```
authnzossnz-worker-1      | Running migrations:
authnzossnz-worker-1      | Traceback (most recent call last):
authnzossnz-worker-1      |   File "/ak-root/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 87, in_execute
authnzossnz-worker-1      |     return self.cursor.execute(sql)
authnzossnz-worker-1      |            ^^^^^^^^^^^^^^^^^^^^^^^^
authnzossnz-worker-1      |   File "/ak-root/venv/lib/python3.11/site-packages/django_prometheus/db/common.py", line 69,in execute
authnzossnz-worker-1      |     return super().execute(*args, **kwargs)
authnzossnz-worker-1      |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
authnzossnz-worker-1      |   File "/ak-root/venv/lib/python3.11/site-packages/psycopg/cursor.py", line 737, in execute
authnzossnz-worker-1      |     raise ex.with_traceback(None)
authnzossnz-worker-1      | psycopg.errors.DuplicateColumn: column "throttling_failure_count" of relation "authentik_stages_authenticator_static_staticdevice" already exists
```
and Authentik is giving a 502 error...
